go.mod

```diff
@@ -20,3 +21,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 )
+
+replace github.com/ammmze/wsman => github.com/jacobweinstock/wsman v0.0.0-20220921012244-df9dee44b0c4
```

Comment:
would like to remove this. Will need https://github.com/ammmze/wsman/pull/1 to be merged first though.